app/src/main/java/com/example/frogweather/data/Main.kt

```diff
@@ -0,0 +1,21 @@
+package com.example.frogweather.data
+
+import com.squareup.moshi.Json
+
+data class Main(
+    val temp: Double = 0.0,
```

Comment:
Same here, and also the name of the object could be different, don't understand what main means